pkg/frame/compressed.go

```diff
@@ -6,7 +6,25 @@ import (
 	"image/jpeg"
 )
 
+// Thank you to https://github.com/filiptc/gorbit/blob/fa87ff39b68a6706306f34c318e0b9a5a3c97110/image/overlay.go#L37-L40 for addMotionDht, dhtMarker, dht, and sosMarker
```

Comment:
licensing checks out 👍

Reply:
Strictly speaking, when OSS licensed code is copied, we should include the full original license statements according to
```
The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.
```
in case of MIT.
https://github.com/filiptc/gorbit/blob/fa87ff39b68a6706306f34c318e0b9a5a3c97110/LICENSE